dvc/output.py

```diff
@@ -982,6 +980,18 @@ def merge(self, ancestor, other):
     def fspath(self):
         return self.path_info.fspath
 
+    @property
+    def is_decorated(self) -> bool:
+        return self.is_metric or self.is_plot
```

Comment:
Not sure I understand why `decorated`, could you elaborate on naming?

Reply:
`decorated` as in bearing extra items like `plots`, `metrics`, etc.
> decorate - **make (something)** look more attractive **by adding extra items or images to it**.